src/picongpu/include/particles/creation/creation.kernel

```diff
@@ -0,0 +1,305 @@
+/**
+ * Copyright 2015-2016 Heiko Burau
```

Comment:
also, as a follow up: pls rebase `ionization.hpp` to use *this* new implementation because code duplication is not a good idea.